<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="Tab1">
      <attribute name="title">
       <string>数据源配置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="9,1" columnstretch="3,7">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="Label">
            <property name="text">
             <string>索引:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="index1">
            <property name="specialValueText">
             <string/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Label_2">
            <property name="text">
             <string>名称:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="name1">
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>数据源别名</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="Label_3">
            <property name="text">
             <string>使能:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="enable1">
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="Label_4">
            <property name="text">
             <string>备注:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="desc1">
            <property name="placeholderText">
             <string>备注信息</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="Label_5">
            <property name="text">
             <string>用户名:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="username">
            <property name="placeholderText">
             <string>OPCUA服务器登录用户名(可为空)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="Label_6">
            <property name="text">
             <string>密码:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="password">
            <property name="placeholderText">
             <string>OPCUA服务器登录密码(可为空)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="Label_7">
            <property name="text">
             <string>地址:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="address">
            <property name="placeholderText">
             <string>OPCUA服务器地址(不可为空)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="connectTest">
            <property name="text">
             <string>连接测试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearDataSource">
            <property name="text">
             <string>清除数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveDataSource">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab2">
      <attribute name="title">
       <string>数据集配置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" rowstretch="9,1" columnstretch="3,7">
       <item row="0" column="1" rowspan="2">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView2">
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="getDataTest">
            <property name="text">
             <string>连接测试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearDataSet">
            <property name="text">
             <string>清除数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveDataSet">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>允许写:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="Label_17">
            <property name="text">
             <string>使能:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="namespaceIndex">
            <property name="placeholderText">
             <string>OPCUA节点--命名空间(不可为空)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>数据源:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QComboBox" name="dataType">
            <item>
             <property name="text">
              <string>Boolean</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Int16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UInt16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Int32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UInt32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Int64</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UInt64</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Float</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Double</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>String</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DateTime</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="Label_18">
            <property name="text">
             <string>备注:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="Label_15">
            <property name="text">
             <string>索引:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="enable2">
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="Label_21">
            <property name="text">
             <string>标识:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="index2">
            <property name="minimum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QComboBox" name="identifierType">
            <item>
             <property name="text">
              <string>String</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Numeric</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ByteString</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>String_Alloc</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Label_16">
            <property name="text">
             <string>名称:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="Label_19">
            <property name="text">
             <string>命名空间:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLineEdit" name="identifier">
            <property name="placeholderText">
             <string>OPCUA节点--标识(不可为空)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="Label_20">
            <property name="text">
             <string>节点类型:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="dataSource">
            <property name="whatsThis">
             <string>当前数据集的数据源</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="name2">
            <property name="placeholderText">
             <string>数据集别名</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>数据类型:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="desc2">
            <property name="placeholderText">
             <string>备注信息</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="writeEnable">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
